src/Task.ts

```diff
@@ -263,18 +266,32 @@ export class Task {
         return li;
     }
 
-    public toString(): string {
-        const recurrenceRule: string = this.recurrenceRule
-            ? ` 🔁 ${this.recurrenceRule.toText()}`
-            : '';
-        const dueDate: string = this.dueDate
-            ? ` 📅 ${this.dueDate.format(Task.dateFormat)}`
-            : '';
-        const doneDate: string = this.doneDate
-            ? ` ✅ ${this.doneDate.format(Task.dateFormat)}`
-            : '';
-
-        return `${this.description}${recurrenceRule}${dueDate}${doneDate}${this.blockLink}`;
+    public toString(layoutOptions?: LayoutOptions): string {
+
+        var taskString = this.description;
```

Comment:
Please always use `let` and `cons` everywhere instead of `var` (another JS quirk).